tests/update_reference_data.sh

```diff
@@ -71,6 +71,7 @@ tests[spe1_thermal_watvisc]="flow spe1 SPE1CASE2_THERMAL_WATVISC"
 tests[spe1_rockcomp]="flow spe1 SPE1CASE2_ROCK2DTR"
 tests[spe1_brine]="flow spe1_brine SPE1CASE1_BRINE"
 tests[spe1_precsalt]="flow spe1_precsalt SPE1CASE1_PRECSALT"
+tests[gas_precsalt]="flow spe1_precsalt GASWATER_VAPWAT_PRECSALT"
```

Comment:
@akva2: Why did this not create/update reference solutions (cf. OPM/opm-tests#792)?  As far as I can tell the directory, file, and case names are all correct.

Reply:
because FILENAME containts .DATA